Sources/HTTP/Message/Message.swift

```diff
@@ -1,10 +1,17 @@
+/// An HTTP message.
```

Comment:
I'd add a little more explanation of a Message being the base of both Request and Response